PD-1151 Update Nextcloud Tutoiral (#2754)
truenas/documentation: content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md

## Before You Begin

Before using SCALE to install the Nextcloud application you need to [create four datasets](#adding-nextcloud-storage) to use as storage for the Nextcloud application.

> If you are creating a user account to manage this application, add the [local administrator]({{< relref "ManageLocalUsersSCALE.md" >}}) account with sudo permissions enabled.
> If creating a new user, add this user to the dataset ACL permissions.

Reviewer: Should this be something like:

```suggestion
If you are creating a user account to manage this application, enable sudo permissions and add this user to the dataset ACL permissions.

If using the [local administrator]({{< relref "ManageLocalUsersSCALE.md" >}}) account, ensure admin sudo permissions are enabled.
```

?

     
   c. Scroll down to the **Nextcloud Postgres Data Storage** option.
      Select **Host Path (Path that already exsits on the system)** in **Type**, then browse to and select the **pgpdata** dataset to populate the **Host Path**.
    
   d. Scroll down to **Nextcloud Postgres Backup Storage**, select **Host Path**, and then enter or browse to the path for the **pgbbackup** dataset.
      When complete, the four datasets for Nextcloud are configured.

    {{< trueimage src="/images/SCALE/Apps/InstallNextcloudStorageConfigPostgressHostPaths.png" alt="Add Nextcloud Storage Volumes" id="Add Nextcloud Storage Volumes" >}}

6. Accept the remaining setting defaults.

7. Scroll up to review the configuration settings and fix any errors or click **Install** to begin the installation.

   The **Installed** screen displays with the **nextcloud** app in the **Deploying** state.
   It changes to **Running** when ready to use.
   Click **Web Portal** on the **Application Info** widget to open the Nextcloud web portal sign-in screen.

{{< trueimage src="/images/SCALE/Apps/NextcloudSignInScreen.png" alt="Nextcloud Sign In Screen" id="Nextcloud Sign In Screen" >}}

## Troubleshooting Tips

> The Nextcloud apps with releases earlier than 2.0.4 are known to have issues. Use the **Upgrade** option in the SCALE UI to update your Nextcloud release to 2.0.4.

Reviewer: ```suggestion
There are known issues with Nextcloud app releases earlier than 2.0.4. Use the **Upgrade** option in the SCALE UI to update your Nextcloud release to 2.0.4.
```

> For more information on known issues, click [here](https://ixsystemsinc.slack.com/archives/C072YK2QZKM/p1715369467541109).

Reviewer: Appears to be an incorrect link here. Maybe you meant to copy a link from slack and accidentally linked to Slack? Readers won't be able to access this.

Author: That is a bad link! I'll go get the Jira ticket link.

Author: Corrected the link.
